ioSPI.yml

```diff
@@ -12,3 +12,5 @@ dependencies:
   - h5py==2.10.0
   - pytorch
   - pillow==8.2.0
+  - numba
```

Comment:
dependencies should be in alphabetic order (it helps users sparsing them + it avoids us putting the same dependency twice)

Additionally, could you transfer these == in >= ? I have just learned that this is good practice for a number of reasons. thank you!

Reply:
ok